templates/cfp/accepted.html

```diff
@@ -13,6 +13,9 @@
           collaborators to appear in the schedule. Any names you give here
           are separate to the one attached to your ticket.</p>
         {{ render_field(form.name, 9) }}
+        <p>What are your preferred pronouns? These will be displayed in the
```

Comment:
"What pronouns should we use?" maybe. Preferred pronouns is generally recommended against.